src/astro/sql/operators/transform.py

```diff
@@ -62,6 +62,14 @@ def my_sql_statement(table1: Table, table2: Table, execution_date) -> Table:
               "exec_date": execution_date
           }
 
+    You can also pass SQL file to the sql parameter in the transform decorator
```

Comment:
@sunank200 do we want to show this feature here?

Reply:
To me this looks weird, and since we have a different operator for this. Maybe we can remove this?